jenkins.sh

```diff
@@ -52,7 +52,7 @@ if [[ ${GIT_BRANCH} != "origin/master" ]]; then
     app spec lib spec config
 fi
 
-GOVUK_CONTENT_SCHEMAS_PATH=tmp/govuk-content-schemas RAILS_ENV=test bundle exec rake
+GOVUK_CONTENT_SCHEMAS_PATH=tmp/govuk-content-schemas RAILS_ENV=test xvfb-run -a bundle exec rake
```

Comment:
From what I read it seems that we would need `xvfb` on any Ubuntu box, but when running on OSX it is not needed because there is the X11 driver.
However, now I'm not sure why I am able to run it in my VM, so we might want to compare our configurations?